cpp/oneapi/dal/algo/covariance/backend/cpu/BUILD

```diff
@@ -0,0 +1,18 @@
+package(default_visibility = ["//visibility:public"])
+load("@onedal//dev/bazel:dal.bzl",
+    "dal_module",
+    "dal_test_suite",
+)
+
+dal_module(
```

Comment:
I am not planning to add a test on this functionality. Because it is not exposed to users. And I do not see a reason to expose this functionality.

So, the potential test should be a black-box test. And for now there is no black box testing in oneDAL. This is not good, but this is the current state. We are not testing every internal kernel that we have, only the APIs that are exposed to the user.

Reply:
We have to start somewhere :) I just added a test to an internal utility function which revealed that we were using it incorrectly before.